src/js/utils/modal.js

```diff
@@ -159,6 +151,22 @@ function toggleShepherdModalClass(currentElement) {
   currentElement.classList.add(classNames.modalTarget);
 }
 
+/**
+ *
+ * @param {SVGElement} element The SVG to set the dimensions on
+ * @param {string} x The x coordinate to set on the element
+ * @param {string} y The y coordinate to set on the element
+ * @param {string} width A string representing the width of the element
+ * @param {string} height A string representing the height of the element
+ * @private
+ */
+function _setElementDimensions(element, x, y, width, height) {
+  element.setAttribute('x', x);
+  element.setAttribute('y', y);
+  element.setAttribute('width', width);
+  element.setAttribute('height', height);
+}
+
```

Comment:
To make this read a bit more explicitly when it's called, we could take an object hash. 

The function could then destructure the properties and it would read pretty much the same here too.

Reply:
Went with an in between option. Using a hash, but not destructuring, and instead using `Object.keys` that way, we can set any attributes we want.